tests/fake_http_server/fake_uptane_server.py

```diff
@@ -8,7 +8,24 @@
 
 class Handler(BaseHTTPRequestHandler):
     def do_GET(self):
-        pass
+        if self.path.startswith("/repo/targets/noerrors"):
```

Comment:
Should this really be for "noerrors"? Won't this get triggered for all the network tests?